### ISSUE TITLE
Extra I/O test failing

### ISSUE THREAD

**Issue by chrissmiller** [France-ioi/taskgrader#36]
I'm having a small bug with a couple tasks pop up again that I mentioned below - it's happening in the Dartmouth/C101/Week1/Comments task, where while testing, the proper solution passes the first test (the one I created) and then fails a second I/O test which just compares the output to a blank file and marks it as a failed test. The extra test does not appear to show up in the raw JSON.

**Comment by mblockelet** [comment]
Indeed, the second test wasn't showing up in the editor's saved data; but I added it and it seems to be working now.

I also noticed that in the bug report #35, you were talking about hints not working for Dartmouth/C101/Week1/loopIt, but I didn't see any hint saved in the editor's data.

Would it maybe mean there are some saving issues?

If so, try to check if there's any error message at the top of the page (we'll make it more obvious in the future) when you save. Thanks!
